src/Twilio/Security/RequestValidator.php

```diff
@@ -154,28 +154,28 @@ private static function addPort(array $parsedUrl): string {
             $port = ($parsedUrl['scheme'] === 'https') ? 443 : 80;
             $parsedUrl['port'] = $port;
         }
-        return self::buildUrl($parsedUrl);
+        return unparse_url($parsedUrl);
     }
+}
 
-    /**
-     * Builds the URL from its parsed component pieces
-     *
-     * @param array $parsedUrl
-     * @return string Full URL
-     */
-    private static function buildUrl(array $parsedUrl): string {
-        $parts = [];
-
-        $parts['scheme'] = isset($parsedUrl['scheme']) ? $parsedUrl['scheme'] . '://' : '';
-        $parts['user'] = $parsedUrl['user'] ?? '';
-        $parts['pass'] = isset($parsedUrl['pass']) ? ':' . $parsedUrl['pass'] : '';
-        $parts['pass'] = ($parts['user'] || $parts['pass']) ? $parts['pass'] . '@' : '';
-        $parts['host'] = $parsedUrl['host'] ?? '';
-        $parts['port'] = isset($parsedUrl['port']) ? ':' . $parsedUrl['port'] : '';
-        $parts['path'] = $parsedUrl['path'] ?? '';
-        $parts['query'] = isset($parsedUrl['query']) ? '?' . $parsedUrl['query'] : '';
-        $parts['fragment'] = isset($parsedUrl['fragment']) ? '#' . $parsedUrl['fragment'] : '';
-
-        return \implode('', $parts);
-    }
+/**
+ * Builds the URL from its parsed component pieces
+ *
+ * @param array $parsedUrl
+ * @return string Full URL
+ */
+function unparse_url(array $parsedUrl): string {
+    $parts = [];
+
+    $parts['scheme'] = isset($parsedUrl['scheme']) ? $parsedUrl['scheme'] . '://' : '';
+    $parts['user'] = $parsedUrl['user'] ?? '';
+    $parts['pass'] = isset($parsedUrl['pass']) ? ':' . $parsedUrl['pass'] : '';
+    $parts['pass'] = ($parts['user'] || $parts['pass']) ? $parts['pass'] . '@' : '';
+    $parts['host'] = $parsedUrl['host'] ?? '';
+    $parts['port'] = isset($parsedUrl['port']) ? ':' . $parsedUrl['port'] : '';
+    $parts['path'] = $parsedUrl['path'] ?? '';
+    $parts['query'] = isset($parsedUrl['query']) ? '?' . $parsedUrl['query'] : '';
+    $parts['fragment'] = isset($parsedUrl['fragment']) ? '#' . $parsedUrl['fragment'] : '';
+
+    return \implode('', $parts);
 }
```

Comment:
Where did this go?

Reply:
Just moved outside the `RequestValidator` class. It's more of a general use url unparser that's now also being used in the `buildUri` function